infra/experimental/SystemSan/SystemSan.cpp

```diff
@@ -68,6 +69,10 @@ const std::string kInjectionError = "Shell injection";
 const std::string kCorruptionError = "Shell corruption";
 // The magic string that we'll use to detect arbitrary file open
 const std::string kFzAbsoluteDirectory = "/fz/";
+// The magic string that we use to disable arbitrary file open
+const std::string kSyssanDisableOpen = "/syssan_open=0";
```

Comment:
Can we omit this for now? I think we want a better mechanism here, and don't want targets to start depending on this already and then later we have to migrate them.

Reply:
Done by resetting to previous commit.

What better mechanism do you see here ?

Reply:
Since this is OSS-Fuzz specific, something in project.yaml or FUZZ_TARGET.options file for this.

Reply:
Okay.
Should I add a command line option to Syssan ? (to disable arbitrary file open)

My proposal is not mutually exclusive, and has the benefit of having finer grained tuning : for instance fuzz_sigpcap target in Suricata can open arbitrary files while processing the sig part of the input, but should not open arbitrary file while processing the pcap part of the input...